core/testsuite/src/test/java/com/blazebit/persistence/testsuite/WindowFunctionTest.java

```diff
@@ -579,6 +575,14 @@ public void testParsedNamedWindowDefinitionWithRange() {
                 .orderByAsc("per.age")
                 ;
 
+        String queryString = criteria.getQueryString();
```

Comment:
Without the port, the frame clause is omitted from the function parameters. So this test shows the problem, and the added assertion will fail without the fix.

Reply:
Good catch, thanks!